compliance/aws/disallowed_regions/aws_disallowed_regions.pt

```diff
@@ -43,11 +49,12 @@ end
 # Authentication
 ###############################################################################
 
-auth "auth_aws", type: "aws" do
-  version "4"
-  service "ec2"
-  access_key cred("AWS_ACCESS_KEY_ID")
-  secret_key cred("AWS_SECRET_ACCESS_KEY")
+#authenticate with AWS
+credentials "auth_aws" do
+  schemes "aws"
```

Comment:
We need to also use the aws_sts scheme.  the schemes should be "aws","aws_sts"

Reply:
Updated the schemes part with "aws","aws_sts"